package-lock.json

```diff
@@ -6363,11 +6363,13 @@
         },
         "balanced-match": {
           "version": "1.0.0",
-          "bundled": true
+          "bundled": true,
```

Comment:
this shouldn't have changed. Are you using node v8.15 ?

Reply:
nope, node v8.11